src/VisualStudio/TestUtilities2/ProjectSystemShim/Framework/TestEnvironment.vb

```diff
@@ -150,18 +149,18 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.ProjectSystemShim.Fr
             Private ReadOnly _fileChangeEx As MockVsFileChangeEx = New MockVsFileChangeEx
 
             Public MockMonitorSelection As IVsMonitorSelection
+            Public VsSolutionMock As MockVsSolution
 
             <ImportingConstructor>
             Public Sub New(exportProvider As Composition.ExportProvider)
                 _exportProvider = exportProvider
+                VsSolutionMock = New MockVsSolution()
             End Sub
 
             Public Function GetService(serviceType As Type) As Object Implements System.IServiceProvider.GetService
                 Select Case serviceType
-                    Case GetType(SVsSolution)
-                        ' Return a loose mock that just is a big no-op
-                        Dim solutionMock As New Mock(Of IVsSolution)(MockBehavior.Loose)
-                        Return solutionMock.Object
+                    Case GetType(IVsSolution), GetType(SVsSolution)
```

Comment:
Something is directly asking *as* `IVsSolution`? That seems fishy.

Reply:
Yes: https://github.com/dotnet/roslyn/pull/35746#discussion_r284809337

Reply:
The request should have been for SVsSolution, not IVsSolution.